prov/verbs/src/fi_verbs.c

```diff
@@ -2480,10 +2483,9 @@ static struct fi_ops_cq fi_ibv_cq_data_ops = {
 	.read = fi_ibv_cq_read_data,
 	.readfrom = fi_no_cq_readfrom,
 	.readerr = fi_ibv_cq_readerr,
-	.write = fi_no_cq_write,
-	.writeerr = fi_no_cq_writeerr,
 	.sread = fi_ibv_cq_sread,
 	.sreadfrom = fi_no_cq_sreadfrom,
+	.signal = fi_no_cq_signal,	/* TODO: write me */
```

Comment:
@a-ilango - I actually could write this function, but I'm hoping that you may be able to help.  :)  @patrickmacarthur needs this functionality for EXS